src/order/category/Frame.lean

```diff
@@ -0,0 +1,58 @@
+/-
+Copyright (c) 2022 Yaël Dillies. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies
+-/
+import order.category.Lattice
+import order.hom.complete_lattice
+
+/-!
+# The category of frames
+
+This file defines `Frame`, the category of frames.
+
+## References
+
+* [nLab, *Frm*][https://ncatlab.org/nlab/show/Frm]
```

Comment:
This still means the wrong thing, see #12385!

Reply:
Uuurgh
```suggestion
* [nLab, *Frm*](https://ncatlab.org/nlab/show/Frm)
```
Thanks for triple-checking